hr_timesheet_default_analytic_account/readme/CONTRIBUTORS.rst

```diff
@@ -1 +1,3 @@
 * Vincent Van Rossem <vincent@coopiteasy.be>
+* Robin Keunen
+* hugues de keyzer
```

Comment:
Peux-tu ajouter les adresses email (et des majuscules) ?

Reply:
i’m happy that you point that out, because it’s unclear to me:
1.  isn’t it a potential spam source to list email addresses there?
2.  what is the convention? `@coopiteasy.be` addresses in readme but personal addresses in commits?
3.  is there a standard sort order?
4.  can i please leave my name in lowercase? i prefer writing it like that (personal quirk 😉).

Reply:
i’ve looked at oca modules, and indeed, most people list their (professional) email address. there is also this syntax (like [here](https://github.com/OCA/hr/tree/14.0/hr_employee_calendar_planning)):
*   [company](https://example.com)
    * givenname surname

some are with and some without email address. this allows to contact the company instead of the individual contributor. what do you think about it?

if the convention is to always use `@coopiteasy.be` addresses, i’ll do it, of course. has this already been decided?

Reply:
i’ve took a look at oca modules, and most people specify indeed their email (often professional) address, and do it like this:
*   givenname surname <[emailaddress@example.com](mailto:emailaddress@example.com)>

i’ve also seen this (like [here](https://github.com/OCA/hr/tree/14.0/hr_employee_calendar_planning)):
*   [company](https://example.com):
    * givenname surname

some without email address, some with.

if people leave the company they work at, their address might not work anymore. and if the goal is to be able to contact the company and not the person, maybe the 2nd syntax is better?

what do you think about it? it would be good to have a uniform company policy. if it’s already decided to list people individually with `@coopiteasy.be` addresses, i’m fine with it, of course.

Reply:
I like the second proposition, we could have a go at it. Maybe we should ask a new applicant to write a script that reads a repo, parses the CONTRIBUTORS.rst and replaces the credits :p